src/main/java/org/folio/dew/repository/BaseFilesStorage.java

```diff
@@ -177,16 +178,23 @@ public String upload(String path, String filename) throws IOException {
    * @throws IOException - if an I/O error occurs
    */
   public String write(String path, byte[] bytes, Map<String, String> headers) throws IOException {
+    byte[] bom = {(byte)0xEF, (byte)0xBB, (byte)0xBF};
```

Comment:
Perhaps it makes sense to make this a constant instead of initializing the array in the method each time?
`private static final byte[] UTF8_BOM = new byte[]{(byte) 0xEF, (byte) 0xBB, (byte) 0xBF};`